src/collapse/dtrace.rs

```diff
@@ -75,6 +88,7 @@ impl From<Options> for Folder {
         Self {
             stack: VecDeque::default(),
             occurrences: HashMap::default(),
+            java_vec: Vec::with_capacity(JAVA_VEC_SIZE),
```

Comment:
Ah, yes, this is a good way to do it! Maybe we should do the same in the perf version rather than relying on `SmallVec`. What do you think @jasonrhansen? You initially chose `SmallVec` in #13.

Reply:
Sounds like I good idea to me!

Reply:
https://github.com/jonhoo/inferno/issues/80